.github/workflows/create_release.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Release'
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get Npm Version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
```

Comment:
Per go/github-actions, we need to pin to a specific commit for third-party actions (here and below)